### DISCUSSION TITLE
Feature request automatic restart option.

### DISCUSSION THREAD

**Discussion by lelun** [dl9rdz/rdz_ttgo_sonde#358] [Q&A] [answered]
Hi i have a feature request for RDZ TTGO SONDE, i have deployed my TTGOs away from where i live and i do not have regular Access to them, hence i cant restart them if there local network drops out or anything else happens with out taking a long drive, and at the next sett of locations they are going to be even more remote.

i would like to request a automatic restart option that restarts the TTGO every lets say 12-72 hours. this would really help me out so i dont have to go and check on them every time one goes down, esspecialy if its a easy fix like rebooting it so it can try to rejoin the local wifi network.

**Comment by DG0MG** [comment]
Set the Wifi mode from “Automatic” to “Client only”. Then it always just looks for the WLAN instead of being an access point itself. If that doesn't help, plug the power supply for the TTGO into a timer that switches off briefly once a week (or so).
I can say that the software runs very stable for me and that no "hands-on" is necessary for a long time. I would actually like to have an "uptime" display - how long has this little thing been running since the last reboot ;)

**Reply by lelun** [reply, marked as answer]
forgot to update this but i found the solution. i ended up setting it in client mode and made sure that the location that its at has one wifi network / partisan thing specifically for the TTGO. and then put it in mode one. (client mode) occasionally power to the TTGO or the wifi goes down but sence its in clientmode it will auto reconnect, and its been running for about half a year now with no intervention. altho i need to visit it to re adjust the antenna and add a filter to it. :)
